### ISSUE TITLE
Download model weights doesn't work for DLC Model Zoo

### ISSUE THREAD

**Issue by Souvik-Mandal-Harvard** [DeepLabCut/DeepLabCut#2057]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I tried the DLC model zoo on one human. Once I got the predicted labels, I tried to download the weights, but when I click the "Download model weights" purple button, nothing happens.

### Operating System

I was using Chrome on a MacOS.

### DeepLabCut version

Model Zoo

### DeepLabCut mode

single animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by mk-mccann** [comment]
I'll bump this as well. I'm trying to download weights for the `full_cheetah` dataset, and I get the following URLError (both on Colab and Windows 10 running DLC 2.2.3):

`URLError: <urlopen error [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond>`

**Comment by MMathisLab** [comment]
hi @mk-mccann yes, we had a server outage, but the weights are now back up; let us know if it continues to be an issue; you will need to run 2.3rc3, or pip install from main branch now though

**Comment by jscanzano** [comment]
Was still getting a `HTTP 404: Not Found` when attempting to download weights for `mouse_pupil_vclose` on DLC 2.2.3, so upgraded to 2.3rc3. Now the model weights download(!) but getting a new error when `Loading....`

`FileNotFoundError: [WinError 3] The system cannot find the path specified: '..\\..\\blobs\\89ce837dde35d07a9d96397e7e9a68657b7c592d7a7ff4c56f88011ecdace3d0' -> 'E:\\Slab_VisNav\\DLC\\eyetrack_only-JSC_SLAB-2022-11-29\\dlc-models\\iteration-0\\eyetrack_onlyNov29-trainset95shuffle1\\train\\models--mwmathis--DeepLabCutModelZoo-mouse_pupil_vclose\\snapshots\\93557f0e74390596a2b9fddaab7628a1cc79fb2d\\DLC_mouse_pupil_vclose_resnet_50_iteration-0_shuffle-1.tar.gz'`

The tar file seems to be missing from its directory. Maybe I'm doing something wrong. Please advise. This is all on Windows 11.

**Comment by evianapetropoulou** [comment]
I would like to report that this is happening with the primate_face model as well. 

More specifically, I changed the primate_face model link in the pretrained_model_urls.yaml file to this: https://huggingface.co/mwmathis/DeepLabCutModelZoo-primate_face/resolve/main/DLC_primate_face_resnet_50_iteration-1_shuffle-1.tar.gz but I am getting HTTP Error 404: Not found in return. 

![image](https://user-images.githubusercontent.com/82345816/208927640-8eca0c4a-285d-42fe-a153-fa21f8b44d3e.png)

I am using DLC 2.2.3 on a Windows 10, Version 21H2 operating system. I tried pip install --upgrade --force-reinstall 'deeplabcut[gui,tf]'==2.3rc3 in my DLC environment but I am getting 'ERROR: Invalid requirement: "'deeplabcut[gui,tf]'==2.3rc3", I do not really know why. 

Thank you in advance for your help!
